### ISSUE TITLE
Accessibility failure in navigation items: default nav, Rayleigh Scattering

### ISSUE THREAD

**Issue by ashleykolodziej** [bu-ist/responsive-framework#296]
**Describe the bug**
https://sites-staging.bu.edu/bucuwip2023/ fails accessibility checks when using default nav and Rayleigh Scattering.

**Screenshots**
<img width="327" alt="Screen Shot 2022-06-21 at 12 07 13 PM" src="https://user-images.githubusercontent.com/1828613/174846527-3131cdd7-006d-47e9-bead-45f77923c281.png">

**Additional context**
This is likely due to the logic in the CSS generation.
